.github/git_rules/comment_responder.py

```
@@ -89,31 +80,76 @@ def respond_to_comment(self):
-                )
+            # 변경된 파일 정보
+            files_info = "변경된 파일들:\n"
+            try:
+                files = list(self.pr.get_files())
+                for file in files:
+                    files_info += f"\n파일: {file.filename} (+{file.additions}/-{file.deletions})\n"
+                    if file.patch:
+                        files_info += f"```diff\n{file.patch}\n```\n"
+                    else:
+                        files_info += "변경사항 없음\n"
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)